test/gov/swagger_validation.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CONTAINERID=/swagger-validator-v2
+VALIDATOR_URL=http://172.18.0.8:8080
+EXCHANGE_URL=http://exchange-api:8080
+ESS_URL=https://raw.githubusercontent.com/open-horizon/edge-sync-service/master/swagger.json
+
+curl -Ss ${EXCHANGE_URL}/v1/api-docs/swagger.json > /tmp/swagger.json
+
+EXCHANGEOUTPUT=$(curl -Ss -d @/tmp/swagger.json -H 'Content-Type:application/json' ${VALIDATOR_URL}/validator/debug)
+
+if [ "$EXCHANGEOUTPUT" == "{}" ]
+then
+	echo -e "No errors in the exchange swagger\n"
+else
+	echo -e "The errors in the exchange swagger are as follows:\n"
+	echo $EXCHANGEOUTPUT
```

Comment:
I think you need to exit 1 here to stop the script if there is a failure. Otherwise, there are so many messages that come out of e2edev that no one will ever notice the error. Or, you could remember the error, continue execution and run the validation on the ESS and then exit 1 from the script if either validation fails.